packages/lesswrong/lib/collections/comments/custom_fields.js

```diff
@@ -4,6 +4,22 @@ import { Comments } from "meteor/example-forum";
 import Users from "meteor/vulcan:users";
 
 Comments.addField([
+  /**
+    Body (Overwriting original schema)
+  */
+  {
+    fieldName: 'body',
+    fieldSchema: {
```

Comment:
You actually only need to specify the field you want to change from the original Vulcan setting. Everything else is kept the same. 

So this would only have to have a single line, with `max:undefined` inside of the fieldSchema